addons/mail/static/src/core/common/mail_guest_model.js

```diff
@@ -102,20 +84,22 @@ export class Persona extends Record {
     previousPresencechannel;
     presenceChannel = fields.Attr(null, {
         compute() {
-            const parts = [
-                "odoo-presence",
-                `${this.type === "partner" ? "res.partner" : "mail.guest"}_${this.id}`,
-            ];
+            if (this.id < 0) {
+                // Some guest have negative ids, like -1
+                return;
+            }
+            const channel = `odoo-presence-mail.guest_${this.id}`;
             if (this.im_status_access_token) {
-                parts.push(this.im_status_access_token);
+                return channel + `-${this.im_status_access_token}`;
             }
-            return parts.join("-");
+            return channel;
         },
     });
     /** @type {boolean} */
     is_public;
+    isAdmin = false;
```

Comment:
Also this was renamed to `is_admin` in master.

I think it's really useless to have them here.